xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemux.h

```diff
@@ -243,29 +243,29 @@ class CDVDDemux
   /*
    * Reset the entire demuxer (same result as closing and opening it)
    */
-  virtual void Reset() = 0;
```

Comment:
I already mentioned that this is wrong. please don't change behaviour in such kind of bulk cosmetic changes.